sitecustomize.py

```diff
@@ -0,0 +1,8 @@
+# Auto-initialize Phoenix/OpenInference across all Python entrypoints
```

Comment:
That runs for _any_ python process, even `pip`. We should avoid that because we only need to initialize phoenix when we start the neuro_san server.